brillig_vm/Cargo.toml

```diff
@@ -13,6 +13,16 @@ repository.workspace = true
 [dependencies]
 acir_field.workspace = true
 serde.workspace = true
+blake2 = "0.10.6"
+sha2 = "0.10.6"
+sha3 = "0.10.6"
+k256 = { version = "0.11.0", features = [
+    "ecdsa",
+    "ecdsa-core",
+    "sha256",
+    "digest",
+    "arithmetic",
+] }
```

Comment:
We should make these workspace dependencies to ensure that Brillig + ACVM match.